mteb/tasks/Clustering/eng/MedrxivClusteringS2S.py

```diff
@@ -1,11 +1,50 @@
 from __future__ import annotations
 
+from mteb.abstasks.AbsTaskClustering import AbsTaskClustering
+from mteb.abstasks.AbsTaskClusteringFast import AbsTaskClusteringFast
 from mteb.abstasks.TaskMetadata import TaskMetadata
 
-from ....abstasks.AbsTaskClustering import AbsTaskClustering
+
+class MedrxivClusteringS2SFast(AbsTaskClusteringFast):
+    metadata = TaskMetadata(
+        name="MedrxivClusteringS2SFast",
+        description="Clustering of titles from medrxiv across 51 categories.",
+        reference="https://api.medrxiv.org/",
+        dataset={
+            "path": "mteb/medrxiv-clustering-s2s",
+            "revision": "14e9407488faf256e41082875434d7884ae58363",
+        },
+        type="Clustering",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="v_measure",
+        date=("2021-01-01", "2022-05-10"),
+        form=["written"],
+        domains=["Academic"],
+        task_subtypes=["Thematic clustering"],
+        license="https://www.medrxiv.org/content/about-medrxiv",
```

Comment:
Please check the license what it is?
Examples:
- license="MIT"
- license="CC-BY-NC-SA-4.0"
- license="Not specified"
- license=None